style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: #f7f7f9;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+.navibar {
+  background: #ffff;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.menu-btn {
+  border: none;
+  background-color: #f7f7f9;
+}
+
+.h-container {
+  background-image: url("./images/header-shapes-mobile.png"), url("./images/header-background-mobile.png");
+  background-repeat: no-repeat, no-repeat;
+  background-size: cover, 100% 100%;
+  padding-top: 114px;
+  padding-bottom: 114px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
```

Comment:
- [ ] Please remove the `background-size` property or set it to 100% as that will fix the background image positioning and size to match the Figma design.
![image](https://user-images.githubusercontent.com/62290545/170348826-2c9dc907-7d26-49f4-8637-7b90fcdce6c6.png)